internal/pwru/btf.go

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+/* Copyright Leon Hwang */
```

Comment:
Do you mind adding a line of `/* Copyright Authors of Cilium */` below, like https://github.com/cilium/pwru/blob/main/internal/pwru/types.go#L3 ?